Sources/PointFree/Pricing.swift

```diff
@@ -536,28 +540,28 @@ private let selectors = (
     CssSelector.id("tab1")
   ),
   content: (
-    CssSelector.id("content0"),
-    CssSelector.id("content1")
+    CssSelector.class("content0"),
+    CssSelector.class("content1")
   )
 )
 
 private func tabStyles(
-  idSelectors: [(input: CssSelector, content: CssSelector)],
-  showTabStyles: Stylesheet = display(.flex)
+  selectors: [(input: CssSelector, content: CssSelector)],
+  showTabStyles: Stylesheet = display(.inherit)
```

Comment:
This is kind of a hack, but works! Just need to make sure the parent node uses the proper display.